etc/WMAgentConfig.py

```diff
@@ -210,6 +210,7 @@
 config.JobSubmitter.jobsPerWorker = 100
 config.JobSubmitter.maxJobsPerPoll = 1000
 config.JobSubmitter.cacheRefreshSize =10000 # minimum size of the cache before refresh. prevent frequent cache update
+config.JobSubmitter.highIOjobSites = [ "T0_CH_CERN", "T2_CH_CERN_AI" ]
```

Comment:
Talking in adhere to python conventions :-), you can start by removing the trailing whitespaces inside the list.